background/get-addon-settings.js

```diff
@@ -296,6 +296,19 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
             settings.popup = newPopupSettingValue;
             madeAnyChanges = madeChangesToAddon = true;
           }
+
+          updatePresetIfMatching(
+            settings,
+            7,
+            {
+              // Old highlight color (blue)
+              primary: "#4d97ff",
```

Comment:
`highlightText` also needs to be changed.

Reply:
And `menuBar` either only on Scratch's default colours or always if we're also migrating Experimental Dark.

Reply:
The migration code I added is probably alright but only as a fallback if none of the presets that changed match exactly. We can discuss it after the other part of the PR is approved by mxmou.

Reply:
> `highlightText` also needs to be changed.

Right. That default has changed too.